src/common/exif.cc

```diff
@@ -161,9 +161,6 @@ void dt_exif_set_exiv2_taglist()
 {
   if(exiv2_taglist) return;
 
-  Exiv2::XmpParser::initialize();
```

Comment:
Are those removals really wanted? I'm don't see the relation with the fix below... Just in case I prefer asking.

Reply:
Yes, the exiv2 initialization is already done in dt_exif_init() which is called before in any case.